docs/docs/module_guides/models/llms/usage_custom.md

```diff
@@ -11,6 +11,10 @@ Below we show a few examples of LLM customization. This includes
 - changing the number of output tokens (for OpenAI, Cohere, or AI21)
 - having more fine-grained control over all parameters for any LLM, from context window to chunk overlap
 
+Note: Some examples below rely on having the data directory present containing the documents to load.
+You can download the example Paul Graham essay from [link](https://raw.githubusercontent.com/run-llama/llama_index/main/docs/docs/examples/data/paul_graham/paul_graham_essay.txt)
+and store it in the `data` directory under the current working directory.
+
```

Comment:
Hmm, I think we'd rather have the link to download present in the notebooks that use them. I believe have included the `wget` command in all the notebooks that require it, but maybe a few of them slipped through the cracks?

```
!mkdir -p 'data/paul_graham/'
!wget 'https://raw.githubusercontent.com/run-llama/llama_index/main/docs/docs/examples/data/paul_graham/paul_graham_essay.txt' -O 'data/paul_graham/paul_graham_essay.txt'
```

Reply:
Ah alright sure! Notebooks do have them, I was going through this reference though.. will close the pr then